### ISSUE TITLE
Using Regularized Segmentation Losses in fully supervised setting

### ISSUE THREAD

**Issue by JunMa11** [dmitrii-marin/adm-seg#2]
Dear @dmitrii-marin ,

Thanks for sharing the code.
Can the Regularized Segmentation Losses be used in fully supervised setting?
If yes, can the performance be further improved?

Best,
Jun

**Comment by dmitrii-marin** [comment]
Yes, it can be done. However, we didn't find it useful to use such
regularized losses in fully supervised setting.

**Comment by JunMa11** [comment]
Hi @dmitrii-marin ,

Thanks for your reply very much. That's interesting. 
Do you know why the regularizer loss is not very useful in fully supervised setting?

**Comment by dmitrii-marin** [comment]
Fully supervised setting gives you perfect information about the labeling, so no extra prior such explored in our paper are needed. That assumes that the ground truth is accurate. Perhaps @meng-tang could provide more insights.

**Comment by meng-tang** [comment]
Regularization prevents over-fitting. However, it seems that at this point no network overfits PASCAL which has more than 10K training images.

**Comment by JunMa11** [comment]
Got it. Thanks for your reply very much.
@dmitrii-marin @meng-tang
